src/Server/Query.elm

```diff
@@ -204,6 +206,19 @@ validateRatio maybeRatio =
             )
 
 
+validateUseNbCycles : Maybe Int -> Maybe ErrorMessage
+validateUseNbCycles maybeUseNbCycles =
+    maybeUseNbCycles
+        |> Maybe.andThen
+            (\int ->
+                if int < 0 || int > 100 then
+                    Just "Un nombre de cycles d'entretien doit être compris entre 0 et 100 inclus."
```

Comment:
Is 100 a random number, or one that we get from the "base impact"? Should we document that somewhere (also maybe in the [openapi.yaml](https://github.com/MTES-MCT/wikicarbone/pull/61#diff-d910ba2ef878f7db0223a966b81c8b3f3b65027bb39e4431bb05140171eece39R256)?)

Reply:
I never had any upper limit specified, so I went with `100` because the max number of cycles defined for products is `45` and I wanted to offer a bit more latitude to inspect variability. I'll add documentation in `openapi.yml` about that.

Reply:
Actually the lower & upper limits were already defined in openapi docs.